homeassistant/components/homeassistant/__init__.py

```diff
@@ -305,21 +317,32 @@ async def async_handle_reload_all(call: ha.ServiceCall) -> None:
             )
 
         services = hass.services.async_services()
-        tasks = [
-            hass.services.async_call(
-                domain, SERVICE_RELOAD, context=call.context, blocking=True
-            )
-            for domain, domain_services in services.items()
-            if domain != "notify" and SERVICE_RELOAD in domain_services
-        ] + [
-            hass.services.async_call(
-                domain, service, context=call.context, blocking=True
-            )
-            for domain, service in {
-                ha.DOMAIN: SERVICE_RELOAD_CORE_CONFIG,
-                "frontend": "reload_themes",
-            }.items()
-        ]
+        tasks = (
+            [
+                hass.services.async_call(
+                    domain, SERVICE_RELOAD, context=call.context, blocking=True
+                )
+                for domain, domain_services in services.items()
+                if domain != "notify" and SERVICE_RELOAD in domain_services
+            ]
+            + [
+                hass.services.async_call(
+                    domain, service, context=call.context, blocking=True
+                )
+                for domain, service in {
+                    ha.DOMAIN: SERVICE_RELOAD_CORE_CONFIG,
```

Comment:
```suggestion
                    ha.DOMAIN: SERVICE_RELOAD_CORE_CONFIG,
                    ha.DOMAIN: SERVICE_RELOAD_CUSTOM_JINJA,
```

Reply:
That results in a duplicate key in the dictionary.  Will switch to a list of tuples here.